src/helpers/columnWidth/columnWidth.spec.js

```diff
@@ -25,4 +25,15 @@ describe('columnWidth', () => {
     const width = columnWidth(mockProps, 'xl');
     expect(width).toEqual(null);
   });
+
+  test('should return "display:none" if breakpoint is explicity set to 0', () => {
```

Comment:
Might also be worth to test the `isHidden` function by itself too. (already exported too)